modules/swift.nix

```diff
@@ -1,26 +1,42 @@
 { pkgs, ... }:
-{
+let swiftc-wrapper = pkgs.stdenv.mkDerivation {
+  name = "swiftc-wrapper";
+  buildInputs = [pkgs.makeWrapper pkgs.swift];
+  src = ./.;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    makeWrapper ${pkgs.swift}/bin/swiftc $out/bin/swiftc \
+      --set PATH ${pkgs.lib.makeBinPath [
```

Comment:
Do you have to put swift on the path here, or is it sufficient to have an empty PATH?

Reply:
It's weird to me that we need to have swift on the path of the swift compiler.

Reply:
I think you are right. I'll try removing it.

Reply:
Probably still want to set PATH though... to empty?

Reply:
Yup. Setting the PATH to empty is needed to make it work.